### PR TITLE
Add new metadata fields to ParticipatoryProcesses

### PR DESCRIPTION
**Pull request by aitorlb**
#### :tophat: What? Why?
- Add new metadata fields to `ParticipatoryProcesses`
- Adapt form and commands
- Modify views (backoffice and public) and add locales (:ca, :es)

#### :pushpin: Related Issues
- Related to #55

#### :clipboard: Subtasks
- [x] Add `CHANGELOG` entry

#### New fields backoffice (large image)
https://drive.google.com/open?id=1YjiHAaOEg7E7yEY36PcO2GyQZ-RydCjG
#### New fields public view
new_metadata_fields_public_view (1)

### CODE REVIEW

File: decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb [hunk 1]
```diff
@@ -79,6 +79,10 @@ def attributes
             announcement: form.announcement,
 
             # Participa added attributes
+            cost: form.cost,
+            has_summary_record: form.has_summary_record,
+            promoting_unit: form.promoting_unit,
+            facilitators: form.facilitators,
             email: form.email,
```
[the remark this answers is not in the document]
Comment (aitorlb) [reply decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb]: You mean at line #91?

[the remark this answers is not in the document]
Comment (tramuntanal) [reply decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb]: I think that the right place is just before L86, `email: form.email,` no?

[the remark this answers is not in the document]
Comment (aitorlb) [reply decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb]: All attributes starting from #82 are unique to `participa`

**Comment by isaacmg410** [comment]
@aitorlb please rebase this branch with master please!
and keep in mind that the fields 

- `decidim_department_id`
- `decidim_type_id`
- ` decidim_theme_id` 

of these files: 

```sh
decidim-process-extended/app/commands/decidim/participatory_processes/admin/create_participatory_process.rb
decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb
decidim-process-extended/app/commands/decidim/participatory_processes/admin/copy_participatory_process.rb
```
 must be removed

File: decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (isaacmg410) [reply decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb]: Should we add these attributes also when copying participatory_processes? I mean, add this fields in `decidim-process-extended/app/commands/decidim/participatory_processes/admin/copy_participatory_process.rb`

[the remark this answers is not in the document]
Comment (aitorlb) [reply decidim-process-extended/app/commands/decidim/participatory_processes/admin/update_participatory_process.rb]: Yes, @isaacmg410 . Thanks, I forgot!

**Pull request merged**